routes/inventory-routes.js

```diff
@@ -0,0 +1,85 @@
+const db = require("../models");
+
+module.exports = function(app){
+
+    // GET full inventory
+    app.get("/api/inventory", function(req,res){
+
+        db.Inventory.findAll({
+            include: [db.Products][db.Seller]
+        }).then(function(dbInventory){
+            res.json(dbInventory);
+        });
+    });
+
+    // GET inventory by seller
+    app.get("/api/inventory/seller/:id", function(req,res){
+
+        db.Inventory.findAll({
+            where: {
+                sellerId: req.params.id
+            },
+            include: [db.Products][db.Seller]
+        }).then(function(dbInventory){
```

Comment:
Prefer arrow functions for short functions without this.
.then((dbInventory) => res.json(dbInventory))